.github/workflows/orch-build-tag-publish-and-run-tests.yml

```diff
@@ -157,6 +157,9 @@ jobs:
       contents: 'read'
       id-token: 'write'
     steps:
+      - name: Determine Branch Name
+        id: branch-names
+        uses: tj-actions/branch-names@v8
```

Comment:
we have many different hand-coded implementations of `init-github-context` across many of our actions. I am choosing to not write another one of those and instead use an out-of-the-box third party implementation. All I need is to properly get the branch name for either a `pull_request` event or a `push` event.